graph/cytoscape/cytoscape.go

```diff
@@ -41,9 +40,14 @@ type NodeData struct {
 	Rate4xx         string          `json:"rate4XX,omitempty"`         // edge aggregate
 	Rate5xx         string          `json:"rate5XX,omitempty"`         // edge aggregate
 	RateOut         string          `json:"rateOut,omitempty"`         // edge aggregate
+	RateTcpTx       string          `json:"rateTcpTx,omitempty"`       // edge aggregate
```

Comment:
Again, Tx/Rx -> Sent/Received

Reply:
Ok, I changed all these names. For now, I'm only using "Sent" names, since I removed "Received" because it's not needed for topology.
Perhaps the "received" rates can be added to the graph in an appender. I still don't know what's better.